scheduler/src/cook/mesos/util.clj

```diff
@@ -150,15 +150,21 @@
 (def ^:const default-job-priority 50)
 
 (defn same-user-task-comparator
-  "Comparator to order same user's tasks"
-  [task1 task2]
-  (letfn [(task->feature-vector [task]
+  "Comparator to order same user's tasks
+
+   If consider-backfilled-jobs? is true, then we treat jobs which were backfilled as being the lowest priority"
+  [consider-backfilled-jobs?]
```

Comment:
I would prefer to have two functions that encapsulate `same-user-task-comparator-penalize-backfill` and `same-user-task-comparator` instead of a boolean because then it forces the reader to figure out what the boolean means.